client/packages/reports/src/DetailView/DetailView.tsx

```diff
@@ -29,21 +34,24 @@ export const DetailView = () => {
     (reportArgsJson && JSON.parse(reportArgsJson.toString())) || undefined;
 
   return !report?.id ? (
-    <BasicSpinner />
```

Comment:
I think this one is ok to be a spinner (basically won't ever happen, unless user tries to edit url ?)

Reply:
That's what I did 😉